packages/admin-ui/src/Button/Button.tsx

```diff
@@ -5,7 +5,7 @@ import { cn, cva, VariantProps, makeDecoratable } from "~/utils";
 const buttonVariants = cva(
     [
         "border-transparent rounded font-sans inline-flex items-center justify-center whitespace-nowrap ring-offset-background transition-colors",
-        "disabled:pointer-events-none",
+        "disabled:cursor-not-allowed",
```

Comment:
I love the not allowed cursor.. but do we need to remove pointer-events-none here? Wouldn't the user still be able to click on the button now?